app/models/risk_trigger_condition.rb

```diff
@@ -0,0 +1,69 @@
+class RiskTriggerCondition < ActiveRecord::Base
+  ALLOWED_FIELDS = %w[
```

Comment:
I'm nervous about this long hardcoded list; it runs the risk of becoming out of sync with reality if our model definitions change.

_Why_ are these the allowed fields? And can we write some kind of test that will warn us if the underlying semantics end up out of sync? (For instance, if this is a comprehensive list of fields on particular models, can we check that `RiskTriggerCondition::ALLOWED_FIELDS` in fact equals `Work.attribute_names` plus `Notice.attribute_names` plus whatever else?)

A comment explaining why these are the fields we care about would also be helpful, in case we need to update it in future.

Reply:
It was just cleaner to have a list of fields that we know that they will be in use. After rethinking this maybe it won't be so problematic for users to select from every possible field. I've updated the code.